src/mame/drivers/a2600.cpp

```diff
@@ -42,6 +114,75 @@ void a2600_base_state::a2600_mem(address_map &map) // 6507 has 13-bit address sp
 }
 
 
+void a2600_pop_state::memory_map(address_map &map) // 6507 has 13-bit address space, 0x0000 - 0x1fff
+{
+	map(0x0000, 0x007f).mirror(0x0f00).rw(m_tia, FUNC(tia_video_device::read), FUNC(tia_video_device::write));
+	map(0x0080, 0x00ff).mirror(0x0d00).ram().share("riot_ram");
+#if USE_NEW_RIOT
+	map(0x0280, 0x029f).mirror(0x0d00).m("riot", FUNC(mos6532_t::io_map));
+#else
+	map(0x0280, 0x029f).mirror(0x0d00).rw("riot", FUNC(riot6532_device::read), FUNC(riot6532_device::write));
+#endif
+	map(0x0800, 0x0800).rw(FUNC(a2600_pop_state::rom_switch_r), FUNC(a2600_pop_state::rom_switch_w));
+	map(0x1000, 0x1fff).rw(m_bankdev, FUNC(address_map_bank_device::read8), FUNC(address_map_bank_device::write8));
```

Comment:
Views are preferred over going through multiple levels of indirection and trampolines for `address_map_bank_device`.

Reply:
changed to memory bank